### PR TITLE
Update sane doc reports automation

### PR DESCRIPTION
**Pull request by agamm**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/18252

## Description
Updating sane-doc-reports.

## Note
The build *will* fail, because I am changing the docker image tag (but make sure it only fails because of that).

### CODE REVIEW

File: Scripts/SaneDocReport/SaneDocReport.yml [hunk 1]
```diff
@@ -2,27 +2,39 @@ args:
...
-dockerimage: demisto/sane-doc-reports:1.0.0.1087
+dockerimage: demisto/sane-doc-reports:1.0.0.1443
 runas: DBotWeakRole
 runonce: false
-deprecated: true
 tests:
-- No Test
\ No newline at end of file
+- No Test
+deprecated: true
```
Comment (meirwah) [inline Scripts/SaneDocReport/SaneDocReport.yml]: why it should be deprecated ?

File: Scripts/SaneDocReport/SaneDocReport.py [hunk 2]
```diff
@@ -1,7 +1,10 @@
-import os
 import demistomock as demisto
 from CommonServerPython import *
 from CommonServerUserPython import *
+import os
+
```
Comment (Itay4) [inline Scripts/SaneDocReport/SaneDocReport.py]: remove empty lines

File: Scripts/SaneDocReport/SaneDocReport.yml [hunk 3]
```diff
@@ -2,27 +2,39 @@ args:
...
+- default: false
+  description: orientation of the report
+  isArray: false
+  name: orientation
+  required: false
+  secret: false
+- default: false
+  description: the paper size of the report
+  isArray: false
+  name: paperSize
```
Comment (Itay4) [inline Scripts/SaneDocReport/SaneDocReport.yml]:
```suggestion
  name: paper_size
```

Comment (Itay4) [inline Scripts/SaneDocReport/SaneDocReport.yml]: add predefined options and default value (A4)

File: Scripts/SaneDocReport/SaneDocReport.py [hunk 4]
```diff
@@ -10,13 +13,20 @@
...
 
-    run('sane.json', OUTPUT_FILE_PATH)
+    run('sane.json', OUTPUT_FILE_PATH, {
+        'orientation': orientation.decode('utf-8', 'ignore'),
+        'paper_size': paper_size.decode('utf-8', 'ignore')
+    })
 
     with open(OUTPUT_FILE_PATH, 'rb') as f:
-        encoded = base64.b64encode(f.read()).decode("utf-8", "ignore")
+        encoded = base64.b64encode(f.read()).decode('utf-8', 'ignore')
```
Comment (Itay4) [inline Scripts/SaneDocReport/SaneDocReport.py]: lets stay consistent with `"` and `'` and use one of them (preferably `'`) in all the code

File: Scripts/SaneDocReport/SaneDocReport.py [hunk 5]
```diff
@@ -10,13 +13,20 @@
 try:
     sane_json_b64 = demisto.args().get('sane_docx_report_base64', '').encode(
         "utf-8")
+    orientation = demisto.args().get('orientation', 'portrait').encode(
+        "utf-8")
```
Comment (Itay4) [inline Scripts/SaneDocReport/SaneDocReport.py]: i think the codec should be provided as an argument with default value of `utf-8`
what do you think ?

File: Scripts/SaneDocReport/SaneDocReport.yml [hunk 6]
```diff
@@ -2,27 +2,39 @@ args:
...
   description: sane report json file contents base64
   isArray: false
-  name: "sane_docx_report_base64"
+  name: sane_docx_report_base64
   required: true
   secret: false
+- default: false
+  description: orientation of the report
+  isArray: false
+  name: orientation
```
Comment (Itay4) [inline Scripts/SaneDocReport/SaneDocReport.yml]: add predefined options and default value (portrait)

**Comment by Itay4** [comment]
@kirbles19 please review the yml file

File: Scripts/SaneDocReport/SaneDocReport.yml (hunk 1 above)
Comment (agamm) [reply Scripts/SaneDocReport/SaneDocReport.yml]: We already talked about this, we didn't want it to be shown as it is an internal automation.

File: Scripts/SaneDocReport/SaneDocReport.yml (hunk 3 above)
Comment (agamm) [reply Scripts/SaneDocReport/SaneDocReport.yml]: I can't change this, this is information that is passed from Golang, @Bars92 can elaborate.

File: Scripts/SaneDocReport/SaneDocReport.py (hunk 5 above)
Comment (agamm) [reply Scripts/SaneDocReport/SaneDocReport.py]: I don't understand, did you mean encoding? and if so again, what do you mean?

**Comment by agamm** [comment]
Passes, https://circleci.com/gh/demisto/content/27379 @glicht @yaakovi

**Pull request merged**